library/std/src/sync/once.rs

```diff
@@ -202,6 +204,20 @@ impl Once {
     /// // once any success happens, we stop propagating the poison
     /// INIT.call_once(|| {});
     /// ```
+    ///
+    /// Poison a [`Once`] by explicitly calling [`OnceState::poison`]:
+    ///
+    /// ```
+    /// #![feature(once_poison_pub)]
+    ///
+    /// use std::sync::Once;
+    ///
+    /// static INIT: Once = Once::new();
+    ///
+    /// // poison the once without panicking
+    /// INIT.call_once_force(|p| p.poison());
+    /// INIT.call_once_force(|p| assert!(p.is_poisoned()));
+    /// ```
```

Comment:
These docs look a bit off, maybe from the rebase? This example would be better in the docs for `poison`, and the heading "Poison a [`Once`] by panicking in a `call_once` closure:" is added twice (once here, once above the existing example).